pkg/resources/collector/templates.go

```diff
@@ -101,6 +102,53 @@ journald/k3s:
   units: [ "k3s" ]
   directory: {{ . }}
 `))
+
+	templateKubeAuditLogs = template.Must(template.New("kubeauditlogsreceiver").Parse(`
+filelog/kubeauditlogs:
+  include: {{ . }}
+  start_at: beginning
+  include_file_path: false
+  include_file_name: false
+  operators:
+  - type: json_parser
+    id: parse-body
+    timestamp:
+      parse_from: attributes.stageTimestamp
+      layout: '%Y-%m-%dT%H:%M:%S.%LZ'
+  - type: add
+    field: attributes.log_type
+    value: controlplane
+  - type: add
+    field: attributes.kubernetes_component
+    value: kubeauditlogs
+  - type: move
+    from: attributes.stage
+    to: resource["k8s.auditlog.stage"]
+  - type: move
+    from: attributes.stageTimestamp
+    to: resource["k8s.auditlog.stage_timestamp"]
+  - type: move
+    from: attributes.level
+    to: resource["k8s.auditlog.level"]
+  - type: move
+    from: attributes.auditID
+    to: resource["k8s.auditlog.audit_id"]
+  - type: move
+    from: attributes.objectRef.resource
+    to: resource["k8s.auditlog.resource"]
```

Comment:
1) Should these 5 fields be indexed?
2) Where do we select which fields are indexed?

Reply:
I think leaving these is fine.  Any fields that are included will get indexed